app/controllers/posts_controller.rb

```diff
@@ -10,4 +10,31 @@ def show
     @post = Post.find(@user_id)
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
```

Comment:
- [ ] **[OPTIONAL]** On the user show page it would be nice to have a button that takes the user to the new post form, instead of having to type in the route every time.

  Generally, it would be nice to have buttons that lead users to respective pages, instead of having to type everything in the address bar.
  
  * Each user on the user index pages should be linked to the user.
  * Each post on the user show page should be a clickable link to the post.
  * The post page should have a button for taking the user to the form,
  * and so one.